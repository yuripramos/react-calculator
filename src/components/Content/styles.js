import styled, { css } from "styled-components";
import { rem, media } from "../../styles/tools";

export const ContentWrapper = styled.div`
  text-align: center;
  padding: ${rem(25)} ${rem(10)};
  min-height: ${rem(250)};
  ${media.md(css`
    min-height: ${rem(300)};
    padding: ${rem(50)} ${rem(10)};
  `)};
`;

export const CalcWrapper = styled.div`
  width: 100%;
  height: ${rem(150)};
  background: #FFFFF5;
`;

export const Button = styled.button`
  width: ${rem(80)};
  height: ${rem(80)};
  border: solid 1px #000;
  background: #cdcdee;
  padding: ${rem(3)};
  margin: ${rem(5)};

    ${({ highlight }) =>
    highlight &&
    css`
      background: blue;
    `};
`;

export const Input = styled.input`
  width: ${rem(355)};
  height: ${rem(40)};
`;
