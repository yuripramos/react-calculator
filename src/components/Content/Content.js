import React from "react";
import { ContentWrapper, Title, SubTitle } from "./styles";
import { Container, Row, Column } from "../../styles/grid";
import Button from "../common/Button";

function Content() {
  return (
    <ContentWrapper>
      <Container>
        <Row>
          <Column>
            <Title>An title</Title>
            <SubTitle>subtitle</SubTitle>
            <Button to={"/forecast"} opacity={1} isCallToAction>
              start
            </Button>
          </Column>
        </Row>
      </Container>
    </ContentWrapper>
  );
}

export default Content;
