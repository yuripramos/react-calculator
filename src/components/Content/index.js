import { connect } from "redux-zero/react";

import Content from "./Content";
import newsActions from "../../actions/News";

export default connect(
  ({ isFilled, FeedArray }) => ({
    isFilled,
    FeedArray,
  }),
  newsActions
)(Content);
