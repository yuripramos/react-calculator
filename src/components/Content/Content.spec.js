import React from "react";

import { shallow } from "enzyme";

import Content from "./Content";

it("should match snapshot", () => {
  expect(shallow(<Content />)).toMatchSnapshot();
});
