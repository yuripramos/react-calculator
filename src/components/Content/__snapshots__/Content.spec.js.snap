// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<styled.div>
  <Container>
    <Row>
      <Column>
        <styled.h1>
          An title
        </styled.h1>
        <styled.h2>
          subtitle
        </styled.h2>
        <Button
          className={null}
          disabled={false}
          isCallToAction={true}
          loading={false}
          onClick={null}
          opacity={1}
          to="/forecast"
          type="button"
          width={null}
        >
          start
        </Button>
      </Column>
    </Row>
  </Container>
</styled.div>
`;
